css/cart.css

```diff
@@ -0,0 +1,102 @@
+@import url(./nav.css);
+@import url(./footer.css);
+
+.wrapper {
+  background-color: var(--color-extra-light-gray);
+}
+
+.cart-wrapper {
+  grid-template-columns: 2fr 1fr;
+  grid-template-areas:
+    "product-cart product-coupon"
+    "product-cart product-summary";
+}
+
+.cart-wrapper button {
+  border-radius: var(--radius-round);
+}
+
+.product-cart {
+  grid-area: product-cart;
+  max-height: calc(46 * var(--size-9));
+  overflow-y: scroll;
+}
+
+.product-coupon {
+  grid-area: product-coupon;
+  max-height: calc(23 * var(--size-9));
+  align-self: start;
+}
+
+.product-summary {
+  grid-area: product-summary;
+  max-height: calc(23 * var(--size-9));
+  align-self: start;
+}
+
+/* Product-cart */
+
+.card__body {

```

Comment:
- Class names here can be made consistent, commonly and majorly you have used is `-`.

Reply:
This was used to overwrite the css of card component